src/main/java/reposense/git/GitShortlog.java

```diff
@@ -36,6 +36,10 @@
                 .collect(Collectors.toList());
     }
 
+    /**
+     * Obtains summarised version of git log command for the date range given by {@code sinceDate}
```

Comment:
Can consider specifying that it runs this command against the repository located at `root`. Something like `Obtains summarised version of git log from the repository at {@code root} for the date range...`

Reply:
Fixed